Nodejs/Product/Nodejs/Snippets/1033/Nodejs/readfile.snippet

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>readfile</Title>
+      <Description>Code Snippet for reading a file using the fs module</Description>
+      <Shortcut>readfile</Shortcut>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>file</ID>
+          <Default>file</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[fs.readFile('$file$', function (error, data) {
+    if (err) throw err;

```

Comment:
This error should be editable as it is in the other snippets otherwise I'm not convinced it's really worth including since re-throwing the error isn't particularly valuable. As a dev, I'd prefer to just use the if snippet afterwards to add the error handling

Also, the function argument is ```error``` but the if statement checks for ```err``` so it will never throw anyways.